dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +416,46 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+dashboard.header.getSummarizedProgressForAnonymousUser = function() {
```

Comment:
It looks like this function is only used locally. As such, it probably makes sense to not have it live on the dashboard.header namespace, but instead just be function getSummarizedProgressForAnonymousUser.

Really, we should also wrap this entire thing in an anonymous function so that local methods like remixProject don't end up in the global namespace, and only functions explicitly declared on our namespace are "exported". Probably out of scope for this PR though.